### ISSUE TITLE
non-root user execution issue on Fedora (25)

### ISSUE THREAD

**Issue by orx57** [mbusb/multibootusb#169]
On **Fedora 25**, a fresh installation of **MultiBootUSB v8.8.0** from the `multibootusb-8.8.0-1.noarch.rpm` package don't start at non-root user.

Please find here the output of `multibootusb-pkexec` after the prompt for the password:

```bash
$ multibootusb-pkexec 
Running multibootusb version 8.8.0 from installed system...
Starting multibootusb GUI...
No protocol specified
QXcbConnection: Could not connect to display :0
/usr/local/bin/multibootusb-pkexec : ligne 22 :  4236 Aborted                 (core dumped)pkexec --disable-internal-agent "/usr/local/bin/multibootusb" "$@"
```

As workaround, I have run `xhost local:root` command as non-root user.
And I have found some suggestion to look at polkit policies side...

Can other information be useful?

**Comment by mbusb** [comment]
Thanks for the feedback.

What is the policy you want to suggest for policy file?

**Comment by fabmazz** [comment]
This also affects me, using arch linux (just built the package from aur, version 9.0.0)
>$ multibootusb-pkexec
Running multibootusb version 9.0.0 from installed system...
Starting multibootusb GUI...
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
qt.qpa.screen: QXcbConnection: Could not connect to display 
Could not connect to any X display.

**Comment by mbusb** [comment]
Welcome any suggestion or patch.

**Comment by CDEguia** [comment]
I have the same issue, but sadly no suggestions as of yet...
Here is my info:

```
$ uname -a
Linux lochos 4.15.4-300.fc27.x86_64 #1 SMP Mon Feb 19 23:31:15 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

$ multibootusb-pkexec
Running multibootusb version 9.1.0 from installed system...
Starting multibootusb GUI...
No protocol specified
qt.qpa.screen: QXcbConnection: Could not connect to display :0
Could not connect to any X display.
```

And the fix:

```
$ xhost local:root
non-network local connections being added to access control list
$ multibootusb-pkexec
```

Or launch from desktop. Both worked flawlessly after that, but xhost local:root must be run after reboot.

Thanks for the software it's working great!!!

**Comment by sallecta** [comment]
> $ xhost local:root
> non-network local connections being added to access control list

Thank you! Your fix worked on KDE neon User Edition 5.17

**Comment by orx57** [comment]
Close this issue because it has been unfortunately stalled for too long with no activity.
